dev/create-release/releaseutils.py

```diff
@@ -49,13 +49,16 @@
     print("Install using 'sudo pip install unidecode'")
     sys.exit(-1)
 
+if sys.version < '3':
+    input = raw_input
```

Comment:
If we can do the opposite, the diff should be only 4 lines though

Reply:
Two downsides to that approach:
1. We stick with the legacy Python syntax which will unnecessarily complicate our lives (and our diffs) [in 18 months](http://pythonclock.org)
2. This approach reduces the linting errors from 10 down to just 2.